package/upgrade/upgrade_node.sh

```diff
@@ -78,19 +81,23 @@ get_running_vm_count()
 {
   local count
 
-  count=$(kubectl get vmi -A -l kubevirt.io/nodeName=$HARVESTER_UPGRADE_NODE_NAME -ojson | jq '.items | length' || true)
+  if [ $1 == "$ALL_NODES" ]; then
```

Comment:
Please create `get_running_vm_count_all_nodes` to get all VMs. Or refactor the function to accept a parameter.